lib/api/unversioned/types.go

```diff
@@ -17,6 +17,7 @@ package unversioned
 // All resources (and resource lists) implement the Resource interface.
 type Resource interface {
 	GetTypeMetadata() TypeMetadata
+	GetResourceMetadata() ResourceMetadata
```

Comment:
The ResourceMetadata interface calls the equivalent method: "GetObjectMetadata()", so we should use the same name here - it's returning the same information.